tests/test_options.py

```diff
@@ -700,16 +700,32 @@ def test_show_default_boolean_flag_name(runner, default, expect):
     assert f"[default: {expect}]" in message
 
 
-def test_show_default_boolean_flag_value(runner):
-    """When a boolean flag only has one opt, it will show the default
-    value, not the opt name.
+def test_show_true_default_boolean_flag_value(runner):
```

Comment:
Seems like this behavior now contradicts the requested change.

Reply:
It seems prudent to alert the user when such a boolean flag has its default set to `True` as it is not the norm.